.editorConfig

```diff
@@ -3,3 +3,9 @@ root = true
 [{Makefile,**.mk}]
 # Use tabs for indentation (Makefiles require tabs)
 indent_style = tab
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
I would add a new line at the end of this file.

Reply:
Yeah i don't know why my VSCode is not respecting that file for this haha